packages/appstore/src/components/loader/trading-app-skeleton/tradingAppSkeleton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './tradingAppSkeleton.scss';
+import { HorizontalDivider } from '../divider/divider';
+
+const TradingAppCardSkeleton = () => (
+    <div className='trading-app-card-loader'>
+        <div className='skeleton-loader trading-app-card-loader__icon' />
+        <div className='trading-app-card-loader__container'>
+            <div className='trading-app-card-loader__container__content'>
+                <div className='trading-app-card-loader__container__content--details'>
+                    <div className='skeleton-loader trading-app-card-loader__container__content--details__title' />
+                    <div className='skeleton-loader trading-app-card-loader__container__content--details__description' />
+                </div>
+                <div className='skeleton-loader trading-app-card-loader__container__content--button' />
```

Comment:
Long classnames and very hard to read. can you pls clean it up or change to smaller names?

Reply:
We need all the classes to locate the boxes in place, will remove the card to make it a bit shorter